libs/ledger/include/ledger/chain/block_coordinator.hpp

```diff
@@ -360,31 +352,32 @@ class BlockCoordinator
   /// }
 
   /// @name Variables relating to POS consensus
-  uint64_t aeon_period_{0};      ///< Periodicity of committee renewal
-  Flag     syncronised_{false};  ///< Flag to signal if this node is synchronised, or catching up
+  Flag syncronised_{false};  ///< Flag to signal if this node is synchronised, or catching up
   /// @}
 
   /// @name Telemetry
   /// @{
-  telemetry::CounterPtr reload_state_count_;
-  telemetry::CounterPtr synchronising_state_count_;
-  telemetry::CounterPtr synchronised_state_count_;
-  telemetry::CounterPtr pre_valid_state_count_;
-  telemetry::CounterPtr wait_tx_state_count_;
-  telemetry::CounterPtr syn_exec_state_count_;
-  telemetry::CounterPtr sch_block_state_count_;
-  telemetry::CounterPtr wait_exec_state_count_;
-  telemetry::CounterPtr post_valid_state_count_;
-  telemetry::CounterPtr pack_block_state_count_;
-  telemetry::CounterPtr new_syn_state_count_;
-  telemetry::CounterPtr new_exec_state_count_;
-  telemetry::CounterPtr new_wait_exec_state_count_;
-  telemetry::CounterPtr proof_search_state_count_;
-  telemetry::CounterPtr transmit_state_count_;
-  telemetry::CounterPtr reset_state_count_;
-  telemetry::CounterPtr executed_block_count_;
-  telemetry::CounterPtr mined_block_count_;
-  telemetry::CounterPtr executed_tx_count_;
+  telemetry::CounterPtr         reload_state_count_;
+  telemetry::CounterPtr         synchronising_state_count_;
+  telemetry::CounterPtr         synchronised_state_count_;
+  telemetry::CounterPtr         pre_valid_state_count_;
+  telemetry::CounterPtr         wait_tx_state_count_;
+  telemetry::CounterPtr         syn_exec_state_count_;
+  telemetry::CounterPtr         sch_block_state_count_;
+  telemetry::CounterPtr         wait_exec_state_count_;
+  telemetry::CounterPtr         post_valid_state_count_;
+  telemetry::CounterPtr         pack_block_state_count_;
+  telemetry::CounterPtr         new_syn_state_count_;
+  telemetry::CounterPtr         new_exec_state_count_;
+  telemetry::CounterPtr         new_wait_exec_state_count_;
+  telemetry::CounterPtr         proof_search_state_count_;
+  telemetry::CounterPtr         transmit_state_count_;
+  telemetry::CounterPtr         reset_state_count_;
+  telemetry::CounterPtr         executed_block_count_;
+  telemetry::CounterPtr         mined_block_count_;
+  telemetry::CounterPtr         executed_tx_count_;
+  telemetry::GaugePtr<uint64_t> block_height_;
+  telemetry::GaugePtr<uint64_t> block_hash_;
```

Comment:
Not sure about this metric. You kind of want something that is actually a number

Reply:
I was just curious when miners are all stuck on a block if it's the same block they're stuck on. Not sure what better way there is to do this.

Reply:
Fair enough - I just think there might be some issues with prom, but lets see